src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog.js

```diff
@@ -109,6 +109,7 @@ import {
     fetchVoltageLevelsListInfos,
 } from '../../../../../services/study/network';
 import { FetchStatus } from '../../../../../services/utils';
+import _ from 'lodash';
```

Comment:
That's the first time we use lodash in gridsuite, maybe use something else ?

Reply:
done